File: single-language-analysis/dataset-files/java/TestSmell156.java
import org.junit.Test;
import static org.junit.Assert.*;

public class TestSmell156 {
    
    @Test
    public void testHelloWorld() {
        Player p = Player.fromDescription("videotestsrc ! fakesink");
        p.play();
        try {
            Thread.sleep(5000);
        } catch (InterruptedException e) {
            e.printStackTrace();
        }
        p.stop();
    }
}

File: single-language-analysis/dataset-files/python/test_smell_156.py
# Original URL: https://github.com/ashwoods/gst-c-crash/blob/44f4fbae488d5e5e9432f4546f025395e89d32ea/tests/test_0.py#L6-L12

async def test_smell_156(Player):
    """Hello world test to verify docker setup"""

    p = Player.from_description("videotestsrc ! fakesink")
    await p.play()
    await asyncio.sleep(5)
    await p.stop()